src/tests/java/htsjdk/samtools/metrics/MetricsFileTest.java

```diff
@@ -156,6 +158,45 @@ public void testWriteMetricsFile() throws IOException, ClassNotFoundException {
         histo.increment(1000, 10981982);
         file.setHistogram(histo);
 
+        //Get same histogram from file using two different methods
```

Comment:
These tests should be moved to their own test methods.

i.e. `testGetHistogram()`, `testGetAllHistograms`, `testWritingAndReadingDifferentSizedHistograms`